tests/accessor_legacy/accessor_api_local_common.h

```diff
@@ -69,17 +69,6 @@ class check_local_accessor_api_methods {
 
       queue.submit([&](sycl::handler &h) {
         auto acc = make_local_accessor_generic<T, dims, mode>(range, h);
-        {
-          /** check size() method
-          */
-          auto accessorCount = acc.size();
-          check_return_type<size_t>(log, accessorCount, "size()");
-          const auto expectedCount = ((dims == 0) ? 1 : count);
-          if (accessorCount != expectedCount) {
-            fail_for_accessor<T, dims, mode, target>(log, typeName,
-                "accessor does not return the correct count");
-          }
-        }
```

Comment:
What is the problem here?

Reply:
This tests the legacy `accessor<target::local>` specialization, which does not have a `size()` member.